### ISSUE TITLE
using string_input_producer with train dataset and validate dataset

### ISSUE THREAD

**Issue by sddi** [tensorflow/tensorflow#16208]
I have two datasets(files), for train and validate respectively. I can successfully load training set thru tf.train.string_input_producer, set num_epochs=5. Then I can iteratively get batch of data to optimize my model.
But, I got stuck when trying to load my validation set by the same way, the program keeps saying "OutOfRange Error" even I didn't set num_epochs in string_input_producer.
Can you supply an example that using string_input_producer  with two or more dataset?
same as the question on stackoverflow: [here](https://stackoverflow.com/questions/37068324/read-big-train-validation-test-datasets-in-tensorflow)
Please help me solve the problem. Thank you very much.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by sddi** [comment]
I have solved the problem, thank you all the same.
